### PR TITLE
Fix button name, state support in "Get started WPF"

### LINKED ISSUE #2123: Confusing instructions for WPF - .Net Core or Framework
Hi,

I find this a bit confusing. Step 3 part 6:
image

There is no Next here, it is Create:

image

Then step 8 tells you to select .Net Framework.
image

Are both Core and Framework supported, the flow of the instructions is a bit unclear, IMHO.

Thanks

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 99f63e2b-08f3-0123-0290-81ed8e9007a8
* Version Independent ID: ddea56f8-1ea4-ac98-1a18-00b107f46a3e
* Content: [Get started with WebView2 in WPF apps - Microsoft Edge Development](https://docs.microsoft.com/en-us/microsoft-edge/webview2/get-started/wpf)
* Content Source: [microsoft-edge/webview2/get-started/wpf.md](https://github.com/MicrosoftDocs/edge-developer/blob/main/microsoft-edge/webview2/get-started/wpf.md)
* Product: **microsoft-edge**
* Technology: **webview**
* GitHub Login: @MSEdgeTeam
* Microsoft Alias: **msedgedevrel**

### PR DESCRIPTION
**Pull request by mikehoffms**
**Rendered article for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/get-started/wpf?branch=pr-en-us-2135 - review the top of article re: creating a project by using VS projection templates.

In major Step 3:
* Minor step 6 changes button name from Next to Create.
* Minor step 8 should specify if Core & Framework are both supported during project creation.  Update: That required major rewrite of top half of article, to cover 2x2 differentiated scenarios that have differing UX:
   * .NET Core project templates.
      * VS 2022.
      * VS 2019.
   * .NET Framework project templates.
      * VS 2022.
      * VS 2019.
That rewrite is done in PR 2174 [which has been merged into the present PR].

old note:
In the Samples Docs PR, see top of the article, which adds clarifying info but not "Core":
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/get-started/wpf?branch=pr-en-us-2069
https://learn.microsoft.com/en-us/microsoft-edge/webview2/get-started/wpf

Fixes #2123 

old note: Do not merge this PR until PR 2174 is merged into it [done].  The present PR [before that merge] only changes [changed] a button name per reported Issue.

### CODE REVIEW

File: microsoft-edge/webview2/get-started/wpf.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Start with a basic desktop project that contains a single main window.
 
    ![The 'Additional information' dialog with a 'Target Framework' dropdown list.](media/wpf-getting-started-create-core-add-info.png)
 
-1. Select **.NET Core 3.1**, **5.0**, **6.0**, or later (not **3.0**).  Then click **Next**.
+1. Select **.NET Core 3.1**, **5.0**, **6.0**, or later (not **3.0**).  Then click the **Create** button.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/get-started/wpf.md:71]: todo: test this

File: microsoft-edge/webview2/get-started/wpf.md [hunk 2]
```diff
@@ -343,6 +343,9 @@ To demonstrate how to use the events, register a handler for `NavigationStarting
 ## Step 8 - Scripting
 
 You can use host apps to inject JavaScript code into WebView2 controls at runtime.  You can task WebView2 to run arbitrary JavaScript or add initialization scripts.  The injected JavaScript applies to all new top-level documents and any child frames until the JavaScript is removed.
+<!-- TODO: are Core and Framework both supported?
+Core and Framework are both supported.
+-->
```
Comment (mikehoffms) [inline microsoft-edge/webview2/get-started/wpf.md:348]: @nishitha-burman - are Core and Framework both supported for scripting here?
Rendered section:
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/get-started/wpf?branch=pr-en-us-2135#step-8---scripting

File: microsoft-edge/webview2/get-started/wpf.md [hunk 3]
```diff
@@ -77,6 +77,7 @@ Start with a basic desktop project that contains a single main window.
 1. Enter values for **Project name** and **Location**.
 
 1. In the **Framework** dropdown list, select **.NET Framework 4.6.2** or later.
+   <!-- TODO: are Core and Framework both supported?  eg clarify a statement like: Core and Framework are both supported. -->
```
Comment (mikehoffms) [inline microsoft-edge/webview2/get-started/wpf.md:80]: @nishitha-burman - are Core and Framework both supported?
Rendered section:
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/get-started/wpf?branch=pr-en-us-2135#step-3---create-a-single-window-webview2-app

Comment (mikehoffms) [inline microsoft-edge/webview2/get-started/wpf.md:80]:
```suggestion
```
@captainbrosset - can you pls delete the hidden comment?  The downstream PR addresses this; both are supported and have different UX, differing between which VS version too.

File: microsoft-edge/webview2/get-started/wpf.md [hunk 4]
```diff
@@ -46,82 +68,168 @@ This tutorial requires Microsoft Visual Studio, not Microsoft Visual Studio Code
 <!-- ====================================================================== -->
 ## Step 3 - Create a single-window WebView2 app
 
-Start with a basic desktop project that contains a single main window.
+Start by creating a basic desktop project that contains a single main window.
 
-1. Open Microsoft Visual Studio. 
+1. Decide whether to create a **.NET Core/5/6** project (newer), or a **WPF App (.NET Framework)** project (older).  For more information, see:
+   * [.NET history](https://learn.microsoft.com/dotnet/core/introduction#net-history) in _What is .NET? Introduction and overview_.
```
Comment (captainbrosset) [inline microsoft-edge/webview2/get-started/wpf.md:74]:
```suggestion
   * [.NET history](/dotnet/core/introduction#net-history) in _What is .NET? Introduction and overview_.
```

Using a relative URL avoids us having to change the domain name in the future when/if it changes.

**Review by nishitha-burman — APPROVED** [review]

**Pull request merged**